commands/connect.js

```diff
@@ -1,24 +1,35 @@
-const chalk = require('chalk');
-const files = require('../lib/files');
-const git = require('../lib/git');
-const getConfig = require('../lib/config').getConfig;
+const chalk = require("chalk");
+const files = require("../lib/files");
+const git = require("../lib/git");
+const getConfig = require("../lib/config").getConfig;
 
-const connect = (username, project, connection, opts) => {
-  console.log(chalk.cyanBright('Connecting remote origin...'));
-  const githubUsername = username || getConfig('github');
+const connect = (username, project, connection, organization, opts) => {
+  console.log(chalk.cyanBright("Connecting remote origin..."));
```

Comment:
to promote standardization, can you create a `TerminalLogger`object in a separate file?

`tLogger` can expose semantically meaningful methods. for example, `tLogger.info('Connecting remote origin')` could output this message using `chalk.cyanBright` color.

Reply:
do this after you integrate typescript

Reply:
ok